README.md

```diff
@@ -45,3 +76,44 @@ include:
 *   Thomas Honeyman (@thomashoneyman) - Infrastructure
 *   Justin Woo (@justinwoo) - Package Sets
 *   Fabrizio Ferrai (@f-f) - Package Sets
+
+Collaborators are encouraged to help out with maintenance in any of the
+following ways (or however else they see fit):
+
+*   Merging PRs if they are ready to be merged (see below)
+*   Managing the issue tracker (labelling, closing, or otherwise tidying up
+    issues)
+*   Discussing proposed features or bug reports
+*   Sending PRs
+*   Reviewing PRs
+
+## Making Decisions
+
+Most decisions which directly concern the language, compiler, or core libraries
+should receive the approval of the core owners. For example, this would apply
+to:
+
+*   Feature requests
+*   Concerns raised during pull request review
+*   Adding new collaborators
+*   Promoting collaborators to owners
+
+For a decision to be made, it should receive the approval of all core owners
+who choose to participate in the discussion. In essence, every core owner has a
+veto (which they are expected to use responsibly).
+
+Not all core owners are required to participate in every discussion -- it is
+expected that most discussions will only involve a subset of the core owners.
```

Comment:
More than anything I think it's just a bar that won't ever be met. If we predicate being a core owner on absolute participation, then we won't have any core owners. I would love to be able to participate meaningfully in every discussion, but it just can't be done.

Reply:
For this I think it would be helpful to breakdown changes in terms of:
* Maintenance (internal compiler changes) 
* Compiler features (public compiler additions)
* Language additions (non-breaking additions to the surface language and semantics)
* Language changes (breaking changes to the surface language and semantics)

Reply:
Yeah, that would definitely make things easier.